Mathlib/Algebra/Ring/Divisibility/Lemmas.lean

```diff
@@ -91,3 +92,32 @@ lemma add_pow_dvd_pow_of_pow_eq_zero_left (hy : y ^ n = 0) :
 end Ring
 
 end Commute
+section CommRing
+
+variable [CommRing R]
+
+lemma dvd_mul_sub_mul_mul_left_of_dvd {p a b c d x y : R}
+    (h1 : p ∣ a * x + b * y) (h2 : p ∣ c * x + d * y) : p ∣ (a * d - b * c) * x := by
+  obtain ⟨k1, hk1⟩ := h1
+  obtain ⟨k2, hk2⟩ := h2
+  refine ⟨d * k1 - b * k2, ?_⟩
```

Comment:
`polyrith` solves this in one line now, as does `  linear_combination d * hk1 - b * hk2`, but neither is imported :-/ I guess this more low-level way works.

Reply:
Yes, I thought about that, but I think this is so easy with `ring` that it is not worth to add a new `import`.